examples/2d/drag_scale_camera.rs

```diff
@@ -0,0 +1,73 @@
+//! How to change the transform and scale of the camera. Affect camera via dragging and scrolling
+
+use bevy::{input::mouse::MouseWheel, prelude::*, sprite::MaterialMesh2dBundle};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(camera_scale)
+        .add_system(camer_drag)
+        .insert_resource(ClearColor(Color::RED))
+        .run();
+}
+
+fn camer_drag(
+    mouse_button_input: Res<Input<MouseButton>>,
+    mut query: Query<(&mut Transform, &mut OrthographicProjection)>,
+
+    winfo: Res<Windows>,
+    mut prev_position: Local<Vec2>,
+) {
+    if let Some(cursor_position) = winfo.get_primary().unwrap().cursor_position() {
+        if mouse_button_input.pressed(MouseButton::Left) {
+            for (mut transf, cam) in query.iter_mut() {
+                // Calculate the delta in mouse position when it has been held down.
+                let delta: Vec2 = (*prev_position - cursor_position) * cam.scale;
+
+                //Applying the motion Delta to the camera's transform
+                *transf = Transform::from_translation(transf.mul_vec3(delta.extend(0.0)));
+            }
+        }
+        *prev_position = cursor_position;
+    }
+}
+fn camera_scale(
+    mut mouse_wheel_events: EventReader<MouseWheel>,
+    mut query: Query<&mut OrthographicProjection, With<Camera>>,
+) {
+    // Query for mouse scroll events, and will apply delta scroll to camera's scale
+    for event in mouse_wheel_events.iter() {
+        for mut que in query.iter_mut() {
```

Comment:
```suggestion
        for mut proj in query.iter_mut() {
```

or projection, but we should be explicit as much as possible in examples